universum/modules/vcs/github_vcs.py

```diff
@@ -17,7 +21,34 @@ def get_time():
     return datetime.datetime.utcnow().replace(microsecond=0).isoformat() + "Z"
 
 
-class GithubMainVcs(ReportObserver, git_vcs.GitMainVcs):
+class GithubToken(Module):
+
+    @staticmethod
+    def define_arguments(argument_parser):
+        parser = argument_parser.get_or_create_group("GitHub", "GitHub repository settings")
+
+        parser.add_argument("--github-integration", "-ght", dest="integration_id", metavar="GITHUB_INTEGRATION",
+                            help="GitHub application ID (see 'general')")
```

Comment:
What does it refer to?
Where is that 'general'?

Reply:
GitHub documentation for Apps. I'd rather fix documentation in next commit, these lines are just copied from previous implementation. I can remove help completely from this commit though for less confusion. (Also applies for the above)

Reply:
I looked there before writing my comment. Could you, please, provide direct link?
Also, considering these are completely new parameters, where you could copy them from?

Reply:
From the old token generation script. Also, I planned on adding links to GitHub documentation later, in docs-update PR

Reply:
> 
> 
> What does it refer to?
> Where is that 'general'?

GitHub user -> Settings -> Developer's settings -> App -> Edit
Will be explained in details later in docs-update PR

Reply:
It is ok to remove help temporarily.
Please note, there is no link to 'general' under App -> Edit.